src/lua/api-gateway/validation/oauth2/userProfileValidator.lua

```diff
@@ -217,6 +218,10 @@ function _M:validateUserProfile()
         if json ~= nil then
 
             local cachingObj = self:extractContextVars(json)
+            if cachingObj.user_country_code == nil then
```

Comment:
This file doesn't have anything to do with storage region. These changes are not necessary here.